api-server/common/models/User-Identity.js

```diff
@@ -41,10 +41,9 @@ export default function(UserIdent) {
       include: 'user'
     };
     // get the email from the auth0 (its expected from social providers)
-    const email =
-      profile && profile.emails && profile.emails[0]
-        ? profile.emails[0].value
-        : '';
+    let val = '' + profile.emails[0].value;
```

Comment:
We still need to check the chain, `profile.emails[0].value` exists, as is currently done below for `email`. what we should do is piggy-back off that.